### ISSUE TITLE
Define how to manage accessibility testing of animated content

### ISSUE THREAD

**Issue by BenBoersma** [cerner/terra-toolkit#373]
## Description

We should define how we want manage accessibility testing during animated content. As brought up by this [PR](https://github.com/cerner/terra-framework/pull/993#discussion_r360134817), we have WDIO tests intermittently failing due to accessibility tests running before an animation is complete. 

## Additional Context / Screenshots

Axe-core also has a similar issue logged https://github.com/dequelabs/axe-core/issues/1866.
